tools/pd-simulator/simulator/node.go

```diff
@@ -151,6 +157,9 @@ func (n *Node) stepHeartBeat() {
 	if n.tick%regionHeartBeatPeriod == 0 {
 		n.regionHeartBeat()
 	}
+	if n.tick%compactionDelayPeriod == 0 {
```

Comment:
IMO, since `stepHeartBeat` mainly handles the heartbeat related logic, this logic should be moved to another place.

Reply:
thanks, done.